data/hai/hai missions.txt

```diff
@@ -1062,6 +1062,362 @@ mission "Hai Honeymoon"
 		payment 100000
 		dialog `Instead of landing in the spaceport where they can be seen, you drop Anaya and Touhar off as close as you can to where they will be staying. The house is a few kilometers away, but they assure you that they can make it on their own. They pay you <payment> and thank you for bringing them this far.`
 
+mission "Hai Solifuge response"
+	landing
+	invisible
+	to offer
+		has "Wanderers Solifuge Recon 2: done"
+	on offer
+		event "Hai Pond Striders released" 10
+		fail
+
+event "Hai Pond Striders released"
+	fleet "Large Hai"
+		add variant 1
+			"Hai Pond Strider" 2
+			"Flea (Pulse)" 8
+			"Flea" 8
+			"Lightning Bug" 2
+		add variant 1
+			"Hai Pond Strider" 3
+			"Flea (Pulse)" 12
+			"Flea" 12
+		add variant 1
+			"Hai Pond Strider"
+			"Flea" 8
+			"Geocoris"
+			"Aphid (Anti-Missile)" 2
+			"Grasshopper"
+		add variant 1
+			"Hai Pond Strider" 2
+			"Flea (Pulse)" 8
+			"Flea" 8
+			"Centipede"
+
+mission "Strider: View of the Pond"
+	description "Deliver a message regarding new Unfettered ships from Ambassador Sayari to Hai elders on <destination>."
+	name "Message Hai-home"
+	landing
+	source "Vara K'chrai"
+	destination "Hai-home"
+	to offer
+		has "Wanderers Solifuge Recon 2: done"
+	on offer
+		conversation
+			`"Ambassador Sayari has a message for the Hai elders about the Solifuge, and needs your help to deliver it to <destination>."`
+				accept
+
+mission "Strider: Have language, will travel"
+	description "Take Ambassador Yashili and her delegation to <destination>."
+	name "Have language, will travel"
+	landing
+	source "Hai-home"
+	destination "Greenwater"
+	passengers 3
+	blocked "The Hai wish to send a delegation to a species that has hull repair technology, and will require 3 bunks for the trip."
+	to offer
+		has "Strider: View of the Pond: done"
+	on offer
+		log `The Hai are working on a medium warship that carries drones in response to the Unfettered, but need help contacting a species with hull repair technology to make it viable.`
+		conversation
+			`As you land the <ship>, you send word that you need to deliver your message from Ambassador Sayari to the Hai elders.  An appointment is arranged, and within hours, you find yourself before them once again.`
+			`The elders tell you how the Solifuge was not entirely a surprise to the Hai government, but that it posed a threat to the Hai public mindset that couldn't be ignored.`
```

Comment:
Starting with even just a bit of dialogue rather than a summary of it would be better IMO.